rollup.config.js

```diff
@@ -0,0 +1,31 @@
+import typescript from '@rollup/plugin-typescript';
+import {terser} from 'rollup-plugin-terser';
+import commonjs from '@rollup/plugin-commonjs';
+
+export default ['cjs', 'esm'].map(format => ({
+	input: 'index.ts',
+	output: {
+		format,
+		dir: format
+	},
+	plugins: [
+		commonjs(),
+		typescript({
+			outDir: format
+		}),
+		terser({
```

Comment:
We can drop terser probably. The reason why it existed in that module is that it had inline tests that I wanted to drop before publishing.